api/payara-bom/pom.xml

```diff
@@ -953,6 +495,9 @@
                 <configuration>
                     <flattenMode>bom</flattenMode>
                     <flattenDependencyMode>all</flattenDependencyMode>
+                    <pomElements>
+                        <dependencyManagement>expand</dependencyManagement>
+                    </pomElements>
```

Comment:
Oh yes, the brutalist explicitness! Will probably save a lot of trouble down the road.

As a both positive and negative of this approach I would note, that this also expands the `<scope>import</scope>` ones resulting in quite long flattened pom, but on the other hand it will be always clear where this comes from.